runtime/compiler/runtime/CMakeLists.txt

```diff
@@ -62,6 +62,7 @@ j9jit_files(
 
 if(JITSERVER_SUPPORT)
 	j9jit_files(
+                runtime/CompileService.cpp
```

Comment:
Could you please align this line with the rest?

Reply:
having a hard time aligning, one moment